src/wolfgame.py

```diff
@@ -3646,35 +3648,36 @@ def transition_day(cli, gameid=0):
     # In general, an event listener < 6 should both stop propagation and prevent default
     # Priority 6 listeners add additional stuff to the default action and should not prevent default
     for victim in vlist:
-        if not revt.dispatch(cli, var, victim):
+        if not revt.dispatch(var, victim):
             continue
-        if (victim in var.ROLES["lycan"] or victim in var.LYCANTHROPES) and victim in revt.data["onlybywolves"] and victim not in var.IMMUNIZED:
-            vrole = get_role(victim)
+        if (victim.nick in var.ROLES["lycan"] or victim.nick in var.LYCANTHROPES) and victim in revt.data["onlybywolves"] and victim.nick not in var.IMMUNIZED:
+            vrole = get_main_role(victim)
             if vrole not in var.WOLFCHAT_ROLES:
                 revt.data["message"].append(messages["new_wolf"])
                 var.EXTRA_WOLVES += 1
-                pm(cli, victim, messages["lycan_turn"])
-                var.LYCAN_ROLES[victim] = vrole
-                change_role(users._get(victim), vrole, "wolf") # FIXME
-                var.ROLES["lycan"].discard(victim) # in the event lycan was a template, we want to ensure it gets purged
-                wolves = list_players(var.WOLFCHAT_ROLES)
+                victim.send(messages["lycan_turn"])
+                var.LYCAN_ROLES[victim.nick] = vrole
+                change_role(victim, vrole, "wolf")
+                var.ROLES["lycan"].discard(victim.nick) # in the event lycan was a template, we want to ensure it gets purged
+                wolves = get_players(var.WOLFCHAT_ROLES)
                 random.shuffle(wolves)
                 wolves.remove(victim)  # remove self from list
-                for i, wolf in enumerate(wolves):
-                    pm(cli, wolf, messages["lycan_wc_notification"].format(victim))
-                    role = get_role(wolf)
+                to_send = []
+                for wolf in wolves:
+                    wolf.send(messages["lycan_wc_notification"].format(victim))
```

Comment:
this message should be queued/bulk-sent (relay_wolfchat_command could result in it being sent twice, so don't use that wrapper here)